### PR TITLE
Auth-supabase-ui

### PR DESCRIPTION
**Pull request by Taneros**
Add workspace boards functionality, sync with db

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,40 @@
...
-        <Router history={newHistory}>
-          <Route exact path="/" component={HomePage} />
-          <Route path="/dashboard" component={DashboardPage} />
-        </Router>
-        <Footer />
-      </div>
+      <AuthProvider supabase={supabase}>
+        <div className="wrapper">
+          <Header />
+          {/* <Router history={newHistory}> */}
```
Comment (IgorLap239) [inline src/App.js:28]: stash comment

File: src/components/Boards/index.jsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+   *
+   **/
+
+  function handleModal() {
+    return setModalShow(true);
+  }
+
+  // function handleModalData(...args) {
+  //   console.log(`handleModalData`, args);
+  // }
```
Comment (IgorLap239) [inline src/components/Boards/index.jsx:43]: stash comment

**Review by IgorLap239 — APPROVED** [review]

**Comment by Taneros** [comment]
> stash comment 

need to check with Alexey why history doesn't work

**Comment by Taneros** [comment]
> stash comment 

will do

**Pull request merged**